libs/common/hexutils.hpp

```diff
@@ -15,6 +15,18 @@
 
 namespace iroha {
 
+  template <typename Container>
+  inline auto hexstringToBytestringSize(Container const &c)
+      -> decltype(c.size()) {
+    return (c.size() + 1) / 2;
```

Comment:
I wonder if strings with odd size were encountered somewhere

Reply:
I would prefere general logic where it is cheap.